xbmc/pvr/PVRManager.cpp

```diff
@@ -667,6 +684,7 @@ void CPVRManager::UnloadComponents()
 {
   m_recordings->Unload();
   m_timers->Unload();
+  m_channelProviders->Unload();
```

Comment:
To keep load/unload symmetric, channel providers should be unloaded after channel groups (or loaded after channel groups).

Reply:
Done